libs/pages/application/src/lib/ui/import-environment-variable-modal/import-environment-variable-modal.tsx

```diff
@@ -0,0 +1,223 @@
+/* eslint-disable-next-line */
+import { Controller, useFormContext } from 'react-hook-form'
+import {
+  Button,
+  ButtonStyle,
+  Dropzone,
+  Icon,
+  IconAwesomeEnum,
+  InputSelectSmall,
+  InputTextSmall,
+  InputToggle,
+  Tooltip,
+} from '@console/shared/ui'
+import { EnvironmentVariableScopeEnum } from 'qovery-typescript-axios'
+import { computeAvailableScope } from '../../utils/compute-available-environment-variable-scope'
+import { DropzoneRootProps } from 'react-dropzone'
+import { EnvironmentVariableSecretOrPublic } from '@console/shared/interfaces'
+import { validateKey, warningMessage } from '../../feature/import-environment-variable-modal-feature/utils/form-check'
+
+export interface ImportEnvironmentVariableModalProps {
+  onSubmit: () => void
+  keys?: string[]
+  availableScopes: EnvironmentVariableScopeEnum[]
+  closeModal: () => void
+  loading: boolean
+  triggerToggleAll: (b: boolean) => void
+  toggleAll: boolean
+  changeScopeForAll: (value: EnvironmentVariableScopeEnum | undefined) => void
+  showDropzone: boolean
+  dropzoneGetRootProps: <T extends DropzoneRootProps>(props?: T) => T
+  dropzoneGetInputProps: <T extends DropzoneRootProps>(props?: T) => T
+  dropzoneIsDragActive: boolean
+  existingVars: EnvironmentVariableSecretOrPublic[]
+  deleteKey: (key: string) => void
+  overwriteEnabled: boolean
+  setOverwriteEnabled: (b: boolean) => void
+}
+
+export function ImportEnvironmentVariableModal(props: ImportEnvironmentVariableModalProps) {
+  const { control, formState, getValues, trigger } = useFormContext()
+  const { keys = [], loading = false, availableScopes = computeAvailableScope(undefined, false) } = props
+
+  // write a regex pattern that rejects spaces
+  const pattern = /^[^\s]+$/
+
+  return (
+    <div className="p-6">
+      <h2 className="h4 text-text-600 mb-6 max-w-sm">Import variables</h2>
+
+      {props.showDropzone ? (
+        <>
+          <div {...props.dropzoneGetRootProps({ className: 'dropzone' })}>
+            <input data-testid="drop-input" {...props.dropzoneGetInputProps()} />
+            <Dropzone isDragActive={props.dropzoneIsDragActive} />
+          </div>
+        </>
+      ) : (
+        <>
+          <div className="flex gap-2 items-center mb-6">
+            <InputToggle
+              dataTestId="overwrite-enabled"
+              value={props.overwriteEnabled}
+              onChange={props.setOverwriteEnabled}
+            />
+            <div className="text-text-500 text-sm font-medium flex items-center gap-2">
+              Enable overwrite
+              <Tooltip content={'If enabled, existing variables will be overwritten.'}>
```

Comment:
You can directly add content="If ..." no { }

Reply:
i don't understand this point amigo :)